project_resource_calendar/views/equipment_calendar_view.xml

```diff
@@ -4,73 +4,86 @@
         <field name="name">calendar.event.form</field>
         <field name="model">calendar.event</field>
         <field name="arch" type="xml">
-            <xpath expr="//button[@class='oe_stat_button pull-right']" position="replace" />
+            <xpath expr="//button[@class='oe_stat_button pull-right']" position="replace"/>
             <xpath expr="//div[@class='oe_title']" position="before">
                 <header>
                     <field name="state" widget="statusbar" statusbar_visible="draft,open,cancelled"/>
                 </header>
             </xpath>
+            <xpath expr="//div[@class='oe_title']" position="before">
+                <div align="right">
+                    <button string="Print Report" type="object" icon="fa-print" name="calendar_event_report"/>
```

Comment:
Where is the calendar_event_report action?

Reply:
@rimbendhaou Add the action  def print_calendar_report(self):